dotCMS/src/main/java/com/dotcms/rest/api/v1/taillog/TailLogResource.java

```diff
@@ -0,0 +1,214 @@
+package com.dotcms.rest.api.v1.taillog;
+
+import static com.dotcms.util.CollectionsUtils.map;
+
+import com.dotcms.repackage.org.apache.commons.io.input.TailerListenerAdapter;
+import com.dotcms.rest.EmptyHttpResponse;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.annotation.NoCache;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.servlets.taillog.Tailer;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.SecurityLogger;
+import com.dotmarketing.util.ThreadUtils;
+import com.dotmarketing.util.UtilMethods;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.io.File;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.Map;
+import java.util.regex.Pattern;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import org.glassfish.jersey.media.sse.EventOutput;
+import org.glassfish.jersey.media.sse.OutboundEvent;
+import org.glassfish.jersey.media.sse.SseFeature;
+import org.glassfish.jersey.server.JSONP;
+
+/**
+ * This resource provides the endpoint used by the LogViewer functionality to display backend server logs
+ * @author nollymarlonga
+ */
+@Path("/v1/tailLog")
+@Tag(name = "TailLog")
+public class TailLogResource {
+
+    @GET
+    @Path("/{fileName}")
+    @JSONP
+    @NoCache
+    @Produces(SseFeature.SERVER_SENT_EVENTS)
+    public final EventOutput getLogs(@Context final HttpServletRequest request,
```

Comment:
Should we support a integer argument  - `linesBack` that reads the file from X lines back, rather than hard coding 5000 or whatever?

Reply:
Done